src/docs/software/using/alphafold.md

```diff
@@ -0,0 +1,296 @@
+    database will take approximately 1 hour to download, you may proceed to
+    the next step while it completes.
+    
+    Although unmodified files are purged from `$SCRATCH` and `$GROUP_SCRATCH`
+    every 90 days, Stanford Research Computing maintains a copy of the
+    AlphaFold 3 databases within the Oak Common Datasets Repository. As you run
+    AlphaFold, you can include a `dcp` in your scripts that compares your
+    `$SCRATCH` databases with those on `$OAK` and automatically copies over
+    any missing files.
+    
+6. Build the AlphaFold 3 Apptainer Image
+
+    Container software like Apptainer allow researchers to easily "contain"
+    or package software and port it between compute platforms. The following
+    commands will allow you to build your own copy of AlphaFold 3 into
+    a `.sif` file, or container image.
+    
+    ```
+    $ sh_dev -m 12GB
+    
+    $ cd $GROUP_HOME/$USER/alphafold3
+    
+    $ wget https://gist.githubusercontent.com/cagancayco/a71ae18ab5044e14526e0c83becaf647/raw/9f6104366d59fa9a96bb92693cfa60930184c981/af3.def
+    
+    $ apptainer build af3.sif af3.def
+    
+    $ exit
+    ```
+    
+    Once you have created the `af3.sif` file, you should put a copy in
+    `$SCRATCH` or `$GROUP_SCRATCH`.
+
+And that's it! Once you have the model parameters, databases, and Apptainer
+image, you are ready to start running AlphaFold 3 on Sherlock.
+
+### Running AlphaFold 3 on Sherlock
+
+1. Set up your input and output directories
+
+    Create directories for AlphaFold 3 inputs and outputs in `$SCRATCH` or
+    `$GROUP_SCRATCH`.
+    
+    ```
+    $ mkdir -p $SCRATCH/af_input
+    $ mkdir -p $SCRATCH/af_output
+    ```
+    
+    The input directory is where you will put the `.json` files containing your input sequence.
+    
+2. Write and submit a batch script for running the AlphaFold 3 pipeline.
+
+    Running AlphaFold 3 can be broken down into two parts: pipeline and
+    inference. The pipeline refers to the genetic sequence search/template 
+    search, and inference refers to predicting structures. GPU's are only 
+    utilized during inference, so we are going to run the pipeline on CPUs.
+    You can use the following batch script as a template. 
+
+    In order to run the pipeline step on a particular sequence, the bash variable
+    `INPUT_JSON` needs to be set to the filename of the input `.json` file
+    you would like to fold, and the `.json` file needs to be placed in the `af_input`
+    directory. In the template below the example input file is `fold_input_2PV7.json`.
+    
+    ```
+    #!/bin/bash
+    #SBATCH --partition=normal
+    #SBATCH --cpus-per-task=8
+    #SBATCH --mem=20G
+    #SBATCH --time=0-01:00:00
+
+    # model and database paths variables
+    MODEL_PARAMS_PATH=$SCRATCH/af3_model
+    DB_PATH=$SCRATCH/af3_db
+    INPUT_JSON=fold_input_2PV7.json
+    
+    # run alphafold3 singularity container
+    singularity run \
```

Comment:
```suggestion
    apptainer run \
```
We should probably only refer to `apptainer` through the page, since that's the official name of the tool now.